src/services/email.js

```diff
@@ -1,32 +1,57 @@
 const EmailTemplates = require('email-templates')
+const path = require('path')
 const { sendMail } = require('~/utils/mailer')
 const { templateList } = require('~/emails')
+
 const {
   gmailCredentials: { user }
 } = require('~/configs/config')
 const { createError } = require('~/utils/errorsHelper')
 const { TEMPLATE_NOT_FOUND } = require('~/consts/errors')
 
-const emailTemplates = new EmailTemplates()
-
+const emailTemplates = new EmailTemplates({
+  views: {
+    root: path.join(__dirname, '../emails'),
+    options: {
+      extension: 'pug'
+    }
+  }
+})
 const emailService = {
   sendEmail: async (email, subject, language, text = {}) => {
+    console.log('Sending email to:', email) // test
+    console.log('Email subject:', subject) // test
```

Comment:
Please, can you remove all `console.log()` and `console.error()`

Reply:
Sorry, it was my check, I shouldn't have added it. Done.